.github/workflows/build.yml

```diff
@@ -131,11 +128,6 @@ jobs:
           binary: "dist/${{ github.event.repository.name }}"
           deb_depends: "openssl"
           rpm_depends: "openssl"
-          config_dir: "packaging/linux"
```

Comment:
I think the action will need to be updated to handle this being nil, I can have a look at this